earth_enterprise/src/fusion/fusionui/AssetDerivedImpl.h

```diff
@@ -127,11 +128,13 @@ template <class Defs, class FinalClass>
 }
 
 template <class Defs, class FinalClass>
-void AssetDerived<Defs, FinalClass>::Init(void) {
+void AssetDerived<Defs, FinalClass>::Init(bool re_init) {
   // defer to final class
   final()->FinalPreInit();
 
-  InstallMainWidget();
+  if (re_init == false) {
+    InstallMainWidget();
+  }
   SetName(saved_edit_request_.assetname);
   SetMeta(saved_edit_request_.meta);
 
```

Comment:
I do not believe main_widget_->Prefill() should be called if we do not call InstallMainWidget(), and this seems to be what is adding duplicate resources in the imagery project in the issue I was looking at.  I need to do some more testing, since this is also tied to behavior of other asset project types.  @tst-lsavoie @tst-jblair